### ISSUE TITLE
Query protocol bloaty

### ISSUE THREAD

**Issue by marcus-pousette** [dao-xyz/peerbit#14]
- AnySearch module can be merge into DQuery and Document and DString and be removed
- Search results from any DQuery should return context (the address)
- Search should allow for querying any state "created at" and "last edited at"

**Comment by marcus-pousette** [comment]
Blocked by #11
